components/observations/ObservationDetailView.tsx

```diff
@@ -214,6 +218,38 @@ export const ObservationCard: React.FunctionComponent<{
   }, [postHog, observation.center_id, observation.id]);
   useFocusEffect(recordAnalytics);
 
+  // currently the back button will leave to the list of obs of your current default center
+  // even if someone shares an observation with you from a different center
+  // the header will still show the current center logo (that likely needs to change as design actually said to replace that with the share icon)
+  // to open in expo: *check url and port, and correct obs ID that exists
+  // example: npx uri-scheme open exp://192.168.1.8:8082/--/observation/866b81db-52b3-4f94-890c-0cae8f162097 --android
+  const url = FormatCenterWebsite(observation.center_id as AvalancheCenterWebsiteSchema) + '/observations/#/view/observations/' + observation.id;
+  const ShareButton = () => {
```

Comment:
Go ahead and label this as a component:

```ts
const ShareButton: React.FunctionComponent = () => {...}
```

Reply:
I think the change is fine since it's the standard in the repo. Just wanted to point out that because of the return type (and I think the naming convention too) it's automatically typed to be a `React.FunctionComponent`!

Reply:
Sorry, not sure what you mean. We almost exclusively use explicit types:

```
$ git grep -E 'React.(FC|FunctionComponent)' | wc -l
132
$ git grep '= () =>' | grep -v use | wc -l
19
```

The inferred return type will be a union of all of the different `return` blocks, so not adding an explicit type will mean it's trivially easy to create whatever mistaken union of types happens to be the set of possible return values and to have one `return` site with a different type than the rest. We're using TypeScript to avoid these kinds of mistakes.

Reply:
I see and that makes sense. I was just pointing out that based on how this is written that it can already be embedded in the JSX as the type is inferred to be `React.JSX.Element` (I was wrong about it being a `React.FunctionComponent`). I understand wanting to maintain the type safety though